apps/remix-ide-e2e/src/tests/terminal.test.ts

```diff
@@ -119,6 +119,26 @@ module.exports = {
       .waitForElementContainsText('*[data-id="terminalJournal"]', 'newOwner', 60000)
       .waitForElementContainsText('*[data-id="terminalJournal"]', '0xd9145CCE52D386f254917e481eB44e9943F39138', 60000)
   },
+  'Run tests using Mocha script and check result logging in the terminal #group4': function (browser: NightwatchBrowser) {
```

Comment:
it would also be interesting to have a test which cover library linking. see https://gist.github.com/yann300/04fb95555dbaa67ab1ea3a23959d4fdc

Reply:
done